libafl/src/bolts/mod.rs

```diff
@@ -110,11 +110,12 @@ pub fn current_time() -> time::Duration {
     time::Duration::from_millis(millis)
 }
 
-/// Given a value v, return a random number using this mixing function
-/// Mixing function <http://mostlymangling.blogspot.com/2018/07/on-mixing-functions-in-fast-splittable.html>
+/// Given a u64 number, return a random number using this mixing function
```

Comment:
It's not a random number, but a hashed number right?

Reply:
hmmm, yes, but it's for making the address number into a more random one.